umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -89,6 +107,15 @@
        (remove #(= "Military Grid Reference System" (:TilingIdentificationSystemName %)))
        seq))
 
+(defn- replace-invalid-format
+  "Replace all the GetData Format in RelatedUrls that doesn't exist in the 1.15.4 schema."
```

Comment:
"Replace all the GetData Format in RelatedUrls that do not exist in the 1.15.4 schema."